website/content/docs/agent/telemetry.mdx

```diff
@@ -89,6 +89,17 @@ These are some metrics emitted that can help you understand the health of your c
 
 **What to look for:** For a healthy cluster, you're looking for a `lastContact` lower than 200ms, `leader` > 0 and `candidate` == 0. Deviations from this might indicate flapping leadership.
 
+### Certificate authority expiration
+
+| Metric Name                            | Description                                                             | Unit    | Type  |
+| :------------------------- | :---------------------------------------------------------------------------------- | :------ | :---- |
+| `consul.mesh.active-root-ca.expiry` | The number of seconds until the root CA expires. Consul updates the value every hour. | seconds | gauge |
+| `consul.mesh.active-signing-ca.expiry` | The number of seconds until the signing CA expires. Consul updates the value every hour. | seconds | gauge |
+
+** Why they're important:** When using an external system, such as Vault, to manage the CA, you should monitor renewals to ensure that certificates do not expire unexpectedly and so that you can rotate as necessary. 
+
+** What to look for:** Rotate your CA prior to expiration by setting a watch based on a rotation schedule.
```

Comment:
The action here depends on whether it's a root or an intermediate CA (similar to the comment above). It might be best to link out to a section in the CA docs that describes this in more detail, rather than trying to include it on the telemetry page (which would just continue to make this page more massive than it already is).

Reply:
The best link I've found is the mesh ca overview...

https://developer.hashicorp.com/consul/docs/connect/ca

Though it doesn't really have a good target to link to which covers these ideas directly. The current version (just pushed) doesn't have the link as I tried to do without due to that.